File: ru.kmorozov.gbd.core/src/main/java/ru/kmorozov/gbd/core/logic/extractors/base/IPostProcessor.java
package ru.kmorozov.gbd.core.logic.extractors.base;

import ru.kmorozov.gbd.core.logic.context.BookContext;

/**
 * Created by km on 09.11.2016.
 */
public interface IPostProcessor extends IUniqueRunnable<BookContext> {

    void setBookContext(BookContext bookContext);

    void make();
}

File: ru.kmorozov.gbd.core/src/main/java/ru/kmorozov/gbd/core/logic/extractors/base/IPostProcessor.kt
package ru.kmorozov.gbd.core.logic.extractors.base

import ru.kmorozov.gbd.core.logic.context.BookContext

/**
 * Created by km on 09.11.2016.
 */
interface IPostProcessor : IUniqueRunnable<BookContext> {

    val withParam: Boolean
        get() = true

    fun make()
}
